ops/services/app_gateway/main.tf

```diff
@@ -30,6 +30,7 @@ resource "azurerm_public_ip" "static_gateway" {
   location            = var.resource_group_location
   allocation_method   = "Static"
   sku                 = "Standard"
+  zones               = ["1", "2", "3"]
```

Comment:
what were the availability zones prior to this change?

Reply:
They were this, except now it's a required field and needs to be called out explicitly.